### ISSUE TITLE
Yoast support

### ISSUE THREAD

**Issue by brettsmason** [eighteen73/pulsar#1]
As DM rely on Yoast for a lot of their activities we should add first class support for it in the theme.
This should include breadcrumbs support where required, along with options to filter markup for Tailwind classes.

**Comment by edjeavons** [comment]
I agree. Perhaps with a config flag (default on) that enables all Yoast-specific code?
